jsonschema-validation.xml

```diff
@@ -172,15 +172,26 @@
 
             <section title="Validation of primitive types and child values">
                 <t>
-                    Two of the primitive types, array and object, allow for child values.  The validation of
-                    the primitive type is considered separately from the validation of child instances.
+                    Two of the primitive types, array and object, allow for child values.
+                    The validation of the primitive type is considered separately from
+                    the validation of child instances.
                 </t>
                 <t>
-                    For arrays, primitive type validation consists of validating restrictions on length.
+                    For arrays, primitive type validation consists of validating
+                    restrictions on length with "minItems" and "maxItems", while
+                    "items" and "additionalItems" control which subschemas apply
+                    to which elements of the array.  In addition, "uniqueItems"
```

Comment:
I find the phrasing
> ...control which subschemas apply to which elements of the array.

a little confusing. It may be just my lack of understanding though. Can you expand on this section please?